confluence/confluence-v1.asciidoc

```diff
@@ -0,0 +1,73 @@
+Confluence Rest Configuration
+----------------------------
+This documentation describs what data the confluence configuration retrives and the endpoints used.
+This is this list of data that rest connector using confluence configuration will crawl.
+
+* spaces 
+* pages (such as Wiki pages), 
+* blog posts,  
+* pages and blog posts comments (indexed as metadata) 
+
+Authentication
+~~~~~~~~~~~~~~~
+Support Basic Authentication with username (atlasian account) and API Token
+
+1. For more info see Documentation here - https://developer.atlassian.com/cloud/confluence/basic-auth-for-rest-apis/ 
+
+2. Check here to create new Token API https://id.atlassian.com/manage/api-tokens
+
+
+Supported Crawl
+~~~~~~~~~~~~~~~
+* Full Crawl supported
+* Re-Crawl supported - `rely on the strayContentDeletion`
+
+
+Rest Connector Variables used
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* ${LW_BATCH_SIZE} - This variable is used to set the `limit` query parameter which controls the number results are return in the response for both `blogpost` and `pages`
+
+* {LW_INDEX_START} - This variable  is used to set the `start` query parameter which use the travers the pagination. Confluence pagination is 0 based. This means the first page number is 0.
```

Comment:
You could add the $ at the beginning of the special variable.

Reply:
You could modify the description to the below:
"This variable  is used to set the `start` query parameter which is used to traverse the pagination. Confluence pagination is 0 index based. This means the first page number is 0"